app/models/requests/JourneyRequest.scala

```diff
@@ -25,4 +25,6 @@ class JourneyRequest[+A](val authenticatedRequest: AuthenticatedRequest[A], val
   val sourceDecId: Option[String] = cacheModel.sourceId
   def isType(`type`: DeclarationType*): Boolean = `type`.contains(declarationType)
   def eori: String = authenticatedRequest.user.eori
+
+  def isDeclarantExporter: Boolean = cacheModel.parties.declarantIsExporter.exists(_.isExporter)
```

Comment:
I don't think this is appropriate place for this method. I'd rather move it to `ExportsDeclaration`

Reply:
Good point. Moved